mne_bids/utils.py

```diff
@@ -55,7 +55,9 @@ def get_kinds(bids_root):
     return kinds
 
 
-def get_entity_vals(bids_root, entity_key):
+def get_entity_vals(bids_root, entity_key, *, ignore_sub=('emptyroom',),
+                    ignore_task=None, ignore_ses=None, ignore_run=None,
+                    ignore_acq=None):
```

Comment:
can you update `whats_new`?

Reply:
Of course! Does this go into the `API` section? Or into the "ordinary" changes section?

Reply:
I would say it would go into the ordinary changes section. Since you're not deprecating anything